requirements.txt

```diff
@@ -15,7 +15,7 @@ python-coveralls
 # -----------------------
 pytest
 pytest-cov
-pytest_cases
+pytest_cases==3.1.1
```

Comment:
@vladdoster I've searched for settings pertaining to pyup in the repository settings but haven't found any (?).  Any tips on where I should be looking?